Framework/Desktop/CocoaLumberjackTests/CocoaLumberjackTests.m

```diff
@@ -0,0 +1,40 @@
+//
+//  CocoaLumberjackTests.m
+//  CocoaLumberjackTests
+//
+//  Created by Alsey Coleman Miller on 2/5/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+
+@interface CocoaLumberjackTests : XCTestCase
+
+@end
+
+@implementation CocoaLumberjackTests
+
+- (void)setUp {
+    [super setUp];
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
+- (void)testExample {
+    // This is an example of a functional test case.
+    XCTAssert(YES, @"Pass");
+}
+
+- (void)testPerformanceExample {
+    // This is an example of a performance test case.
+    [self measureBlock:^{
+        // Put the code you want to measure the time of here.
+    }];
+}
+
+@end
```

Comment:
I know, it's just an obsession of mine. I actually want to make sure that when we write the tests, we don't find stuff that needs to be refactored (as it usually happens), so we can indeed include all the breaking changes in 2.0.0